### DISCUSSION TITLE
[Question] Custom module: how to use the standard admin template ?

### DISCUSSION THREAD

**Discussion by tchapi** [directus/directus#3041] [Archive]
Hi,

I would like to reuse the standard admin template (with the sidebar, notifications, etc) in my custom module. How would that work ?
I guess I need to have the source components  / views somewhere in my custom module `package.json`, in order to be able to import what I need in my `module.vue` file but I can't get my head around how to do it properly

Any tips ?
Thanks !

**Comment by rijkvanzanten** [comment]
Heya! You can render your module in a `<private-view>` component 🙂 This component is available globally

```html
<template>
  <private-view title="My Custom Module">
    Your amazing content here
  </private-view>
</template>
```

**Comment by tchapi** [comment]
Fantastic !!

**Comment by MarcusMBX** [comment]
Helped me aswell, thank you.
